### ISSUE TITLE
Create examples and templates for elements of the standard

### ISSUE THREAD

**Issue by ericherman** [publiccodenet/about#414]
What should a README contain?
Link at least two examples of good README files.

What should a CONTRIBUTING file contain?
Link 2+ examples.

Let's do the straight-forward items first, then make separate issues for elements which are more difficult.

**Comment by ericherman** [comment]
@ElenaFdR requested that we be on the lookout for good examples of architecture descriptions/diagrams, especially from public projects.

Having multiple examples means we could show a range from "best of class" to "minimal, but good enough".

**Comment by Ainali** [comment]
There might be some useable examples in this category: https://commons.wikimedia.org/wiki/Category:Software_architecture

**Comment by ElenaFdR** [comment]
Expand with: 
- examples of governance docs for thriving communities

**Comment by bvhme** [comment]
I made for Amsterdam, as a reference:

* [Guide on writing a README](https://github.com/Amsterdam/amsterdam.github.io/blob/master/guides/write-a-readme.md)
* [Contributing](https://github.com/Amsterdam/amsterdam.github.io/blob/master/CONTRIBUTING.md)

**Comment by Ainali** [comment]
I was thinking a bit on where to add these and perhaps we could start by creating a resource from the related actions in in the new page product-assets-for-early-incubation.md that explains them a bit better and have these as linked examples. If needed they can later also have guides for more detailed how-to's when we have more material and developed thoughts for them.

**Comment by clausmullie** [comment]
This issue starts from the standard for public code, and then tries to identify if linked resources as recommended reading or separate assets can be useful. Should this issue then be in the standard repo?

**Comment by Ainali** [comment]
> Should this issue then be in the standard repo?

It's a good question, and I guess it depends on how extensive this collection should be. The Standard should probably be a quite "clean" repository that only includes the actual standard itself (and basic metadata and documentation for the repository). So if it's just a one-liner in Further reading yes. But more extensive "add-on" guides should live somewhere else. Perhaps on about, or if it is really comprehensive, in a repository of its own.
And when that has been created, it should probably be linked from the Standard as further reading.

**Comment by ElenaFdR** [comment]
> Should this issue then be in the standard repo?

Nope. I saw this as being explicitly a stewardship resource - so that stewards helping a community meet the Standard have a database of useful examples to share with codebase communities.

**Comment by clausmullie** [comment]
Perhaps my question is this:

1) Having the standard as a "clean" repository made sense previously, as it was an asset we develop and maintain. So it's the core ingredients, with the 'messy implementation stuff' on our about repo.

2) However, in light of our [recent community call](https://github.com/publiccodenet/blog/pull/174) in which we discuss making the standard more replicable within the community, the discussion includes having localised flavours, localised implementations, example repositories and so on.

3) Where then, should this activity take place? If the standard repo remains 'clean' - and we host our own 'implementation tools' on about, should community members then also keep their assets in their own spaces/places? Or we do create a space for community members to also contribute their implementations (ie 'the OS2 kickstarted guide to a new repo) and 'messy assets' to the standard repository for other community members to find? In case of the former, me might want to start a list of community implementations?

**Comment by Ainali** [comment]
> 3\. Where then, should this activity take place?

My first thought is that a "flavor" or "localized implementation" would be a downstream repository, nothing that affects this repository at all. This has the advantage that anyone can set it up at anytime and without their adaptations affecting any other user of the standard.

**Comment by Ainali** [comment]
I think that everything discussed in this issue should be integrated in the implementation guide we are creating, see publiccodenet/projects#112.

**Comment by ericherman** [comment]
As we're working with https://github.com/diggsweden/open-source-project-template and https://github.com/OS2offdig/OS2produkt/ to help refine good templates for essential files, I think this issue should change focus: there is still utility in linking some good examples, in particular two examples for each showing "minimum to meet the bar" and "best of class" real-world examples. Perhaps a title change is in order?
